src/Components/FitnessPage.js

```diff
@@ -0,0 +1,280 @@
+              </TouchableOpacity>
+            )}
+          </View>
+          <Text style={styles.title}>{title}</Text>
+          <View>
+            {!isLastStep && (
+              <TouchableOpacity
+                onPress={() => {
+                  carouselRef.current.snapToNext();
+                }}
+              >
+                <MaterialCommunityIcons name="chevron-double-right" color={theme.COLOR_PRIMARY} size={26} />
+              </TouchableOpacity>
+            )}
+          </View>
+        </View>
+        <View style={styles.animation}>
+          <Animation widthRatio={1} heightRatio={1 / 2} animation={new Drill(animation)} />
+        </View>
+        <Text style={styles.instruction}>{instruction}</Text>
+      </>
+    );
+  };
+
+  return (
+    <View style={styles.container}>
+      {activeIndex === drill.steps.length && renderFinish()}
+      {currentStep?.animation && !currentStep?.vimeoId && renderAnimation(currentStep)}
+      {currentStep?.vimeoId && renderVimeo(currentStep)}
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  containerFinish: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: 20,
+  },
+  redoMessage: {
+    fontSize: theme.FONT_SIZE_MEDIUM,
+    color: theme.COLOR_PRIMARY,
+    fontWeight: 'bold',
+    marginVertical: 50,
+  },
+  redoButton: {
+    width: 80,
+    height: 80,
+  },
+  step: {
+    flexDirection: 'row',
+    padding: 20,
+    backgroundColor: theme.COLOR_PRIMARY,
+  },
+  doneAnimation: {
+    flex: 1,
+  },
+  stepTitle: {
+    fontSize: theme.FONT_SIZE_LARGE,
+    color: theme.COLOR_PRIMARY_LIGHT,
+  },
+  stepTitleCurrent: {
+    fontWeight: 'bold',
+    color: theme.COLOR_PRIMARY,
+  },
+  stepTitleDone: {
+    color: theme.COLOR_SECONDARY,
+  },
+  separator: {
+    height: 15,
+    borderRightWidth: 1,
+    borderRightColor: theme.COLOR_PRIMARY_LIGHT,
+  },
+  buttonNext: {
+    position: 'absolute',
+    right: 0,
+  },
+  subWrapper: {
+    flexGrow: 0,
+    flexShrink: 0.7,
+    alignItems: 'center',
+    flexDirection: 'row',
+    marginRight: '10%',
+  },
+  instruction: {
+    marginTop: 20,
+    marginBottom: 20,
+    marginLeft: 20,
+    fontSize: theme.FONT_SIZE_MEDIUM,
+    color: theme.COLOR_PRIMARY,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  videoAlone: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    flex: 1,
+  },
+  line: {
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'space-around',
+    marginTop: 15,
+  },
+  title: {
+    fontSize: theme.FONT_SIZE_LARGE,
+    color: theme.COLOR_PRIMARY,
+    fontWeight: 'bold',
+    textAlign: 'center',
+  },
+  animation: { flex: 9 },
+  videoMultiple: {
+    height: 250,
+  },
+  scrollList: {
+    height: '100%',
+  },
+  allPage: {
+    height: '100%',
+    backgroundColor: theme.COLOR_PRIMARY,
```

Comment:
Is there a reason why we invert colors on this specific page?

Reply:
I feel it's nicer to show the difference between the page and I like the design